README.md

```diff
@@ -10,10 +10,10 @@ A central platform for hosting Ubuntu's various documentation portals.
 
 First [install Docker](https://www.docker.com/community-edition#/download). Linux users may need to add their user to the `docker` group.
 
-Then install [git-repo](https://github.com/esrlabs/git-repo) and [documentation-builder](https://github.com/canonicalltd/documentation-builder). E.g.:
```

Comment:
This file looks weird. I think the ```````` code blocks need to be moved

Reply:
See: https://github.com/WillMoggridge/docs.ubuntu.com/blob/691e1f7ec439365bcc78c7609327c237d02326f1/README.md

Reply:
Pushed a change now, think it fixes it